nipype/interfaces/tests/test_io.py

```diff
@@ -25,6 +26,24 @@
 except:
     noboto = True
 
+# Check for boto3
+noboto3 = False
+try:
+    import boto3
+    from botocore.utils import fix_s3_host
+except:
```

Comment:
Avoid "try, catch all" pattern and specify the class of the expected exception (ImportError in this case). It will make debugging in the future easier.